sda-download/.github/integration/tests/common/80_check_reencrypt.sh

```diff
@@ -18,16 +18,10 @@ dataset="https://doi.example/ty009.sfrrss/600.45asasga"
 file="dummy_data"
 expected_size=1048605
 
-# Download unencrypted full file,  check file size
-curl --cacert certs/ca.pem -H "Authorization: Bearer $token" "https://localhost:8443/s3/$dataset/$file" --output full1.bam
-
-if [ ! -f "full1.bam" ]; then
```

Comment:
I think it is better to keep this checking since otherwise it will reported as "Incorrect file size" later.

Reply:
OK. It makes sense to me. We can removed it for easier debugging.